wiki/Home.md

```
@@ -0,0 +1,11 @@
+Welcome to the NullAway wiki!  We currently have documentation on the following topics:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)